docs/index.asciidoc

```diff
@@ -523,6 +523,8 @@ NOTE: Updating the pattern will require the index template to be rewritten
 
 NOTE: The pattern must finish with a dash and a number that will be automatically incremented when indices rollover.
 
+NOTE: The patter number is a string zero-padded 6 digits wide, see {ref}/indices-rollover-index.html#rollover-index-api-path-params[Rollover path parameters API docs] for details
```

Comment:
*pattern

Reply:
Thanks @pujithkurunji to point it out, fixed :-)